response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -75,15 +76,19 @@ def update_event(collection_exercise_id, tag, timestamp):
         if response.status_code == 400:
             logger.warning('Bad request updating event', collection_exercise_id=collection_exercise_id,
                            tag=tag, timestamp=formatted_timestamp, status=response.status_code)
-            return False
+            response_content = response.content.decode("utf-8")
+            response_json = json.loads(response_content)
+            logger.error('got error', message=response_json['error']['message'])
```

Comment:
Should say something else besides `got error`

Reply:
ill make it validation failed, it's only a preface to the actual message anyways